src/main/kotlin/racingcarWinner/domain/Car.kt

```diff
@@ -1,4 +1,6 @@
-package racingcarWinner.core
+package racingcarWinner.domain
+
+import racingcarWinner.util.MessageCode
 
 class Car(
```

Comment:
```suggestion
class Car(
    carName: String
) {
    private var moveStep: Int = DEFAULT_MOVE_STEP
```

moveStep이 `private var`로 되어있지 않아 외부에서 `moveStep = 3` 이런식으로 값을 변경할 수 있게 됩니다.
`private var` 로 변경해주어 외부에서 값을 변경할수 없도록 해주세요. 

moveStep, carName 은 getter 메서드를 생성하여 값을 이용할 수 있도록 하면 될 것 같습니다.